enlighten/measurement/MeasurementFactory.py

```diff
@@ -123,7 +122,7 @@ def create_thumbnail(self, measurement, is_collapsed=False):
             colors          = self.colors,
             stylesheets     = self.stylesheets,
             is_collapsed    = is_collapsed,
-            view            = measurement.technique,
```

Comment:
I feel like I'm seeing some repeated diffs. Same with `if proc else ""`. 

Do you `merge main` or `merge bug-fix` often enough?

Reply:
Possibly not often enough.  Just did, though.